_posts/2020-10-27-a-closer-look-at-the-interpreter.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "A closer look at the interpreter"
+tags: BEAM erts jit
+author: John Högberg
+---
+
+In my [previous post] we had a look at BEAM, and now that we're more familiar
+with it it's time for us to look at the reference implementation: the
+interpreter.
+
+The interpreter can be thought of as an endless loop that looks at the current
+instruction, executes it, and then moves on to the next one.
+
+In normal builds our code is laid out as [directly threaded code], where each
+instruction consists of the _machine code address_ of its handler, followed by
```

Comment:
Maybe better with :  "instruction consists of the _machine code address_ pointing to the implementation of the instruction"